lib/jekyll-github-metadata/repository_finder.rb

```diff
@@ -46,15 +48,19 @@ def nwo_from_config
       end
 
       def git_exe_path
+        exts = (ENV["PATHEXT"] || "").split(File::PATH_SEPARATOR).push("")
+        cmds = exts.map { |ext| "git#{ext}" }
         ENV["PATH"].to_s
           .split(File::PATH_SEPARATOR)
-          .map { |path| File.join(path, "git") }
-          .find { |path| File.exist?(path) }
+          .map { |path| cmds.map { |cmd| File.join(path, cmd) } }
+          .flatten
+          .find { |path| File.executable?(path) && !File.directory?(path) }
       end
 
       def git_remotes
         return [] if git_exe_path.nil?
-        `#{git_exe_path} remote --verbose`.to_s.strip.split("\n")
+        output, _status = Open3.capture2(git_exe_path, "remote", "--verbose")
```

Comment:
@parkr @benbalter  Can we drop support for Jekyll versions `< 3.4.0` and use `Jekyll::Utils::Exec.run` instead of using `Open3` directly..?

Reply:
Good idea, as GitHub pages uses currently `3.7.4` it shouldn't be a problem.